ams/server/src/main/java/com/netease/arctic/server/optimizing/TaskRuntime.java

```diff
@@ -108,7 +110,9 @@ void reset() {
     invokeConsisitency(
         () -> {
           statusMachine.accept(Status.PLANNED);
-          doAs(OptimizingMapper.class, mapper -> mapper.updateTaskStatus(this, Status.PLANNED));
+          startTime = ArcticServiceConstants.INVALID_TIME;
+          endTime = ArcticServiceConstants.INVALID_TIME;
+          persistTaskRuntime(this);
```

Comment:
`reset()` should reset all run fields `costTime`, `token`, `threadId`, `failReason`.

Reply:
Make sense to me.